### PR TITLE
Embed HTML code from file

### PR DESCRIPTION
**Pull request by rail**
* Use Go 1.16's "embed" functionality to use an external template file
  instead of having it inline.
* Bump `go mod` to go 1.16
* Run `go mod tidy`

---
This change is [Reviewable](https://reviewable.io/reviews/cockroachdb/backboard/11)

### CODE REVIEW

File: server.go [hunk 1]
```diff
@@ -12,281 +13,9 @@ import (
...
-        </tr>
-    {{end}}
-    </tbody>
-</table>
-<div id="backport-command" style="display: none">
-	<span></span>
-</div>
-</body>
-</html>`))
+//go:embed index.gohtml
```
Comment (knz) [inline server.go:16]: The advantage of naming this file e.g. `index.tmpl.html` or `index.go.html` is that we can then load it up in a html editor and get syntax highlighting etc for free.

Comment (rail) [reply server.go:16]: Good point. I'll rename it. Thank you.

Comment (rail) [reply server.go]: Done

**Review by rafiss — APPROVED** [review]
this looks good! thank you

one nit: normally i'd say you should rebase your change on `master` then force-push rather than merging `master` into your PR. but this repo isn't used that much, so doesn't seem like a huge deal to me!

**Comment by rail** [comment]
> one nit: normally i'd say you should rebase your change on master then force-push rather than merging master into your PR. but this repo isn't used that much, so doesn't seem like a huge deal to me!

Totally agree. In this case I was lazy enough and used GitHub's button to merge from master in order to get CI checks.
The actual commit will a Squash and merge commit, so there will no nonlinear history.
image

**Pull request merged**